week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var perstudent = Math.floor(numberOfStudents);
+var permentors = Math.floor(numberOfMentors);
+console.log(perstudent + "%");
+console.log(permentors + "%");
```

Comment:
Hi Deepa, great effort, I think you should use the Math.round instead of the Math.floor because Math.round will help in rounding up the number to the nearest whole number. and don't forget in other to get the right percentage you need to times by 100 and then divide by the total number. Great effort you deserve some accolade

Reply:
Thank you Clement .I have corrected this mistake.Thanks for the observation.Appreciate it.

Reply:
yes i am agree with Clements

Reply:
thanks khan